com.unity.shadergraph/Editor/Resources/Styles/GraphSubWindow.uss

```diff
@@ -0,0 +1,127 @@
+﻿.unity-label {
+    padding-top: 5px;
+    padding-left: 2px;
+    padding-right: 2px;
+    padding-bottom: 2px;
+    margin-left: 4px;
+    margin-top: 2px;
+    margin-right: 4px;
+    margin-bottom: 2px;
+}
+
+.GraphSubWindow {
+    position:absolute;
+    border-left-width: 1px;
+    border-top-width: 1px;
+    border-right-width: 1px;
+    border-bottom-width: 1px;
+    border-radius: 5px;
+    flex-direction: column;
+    background-color: #2b2b2b;
+    border-color: #191919;
+    min-width: 100px;
+    min-height: 100px;
+    width: 300px;
+    height: 300px;
+}
+
+.GraphSubWindow.windowed {
+    position: relative;
+    padding-top: 0;
+    flex: auto;
+    border-left-width: 0;
+    border-top-width: 0;
+    border-right-width: 0;
+    border-bottom-width: 0;
+    border-radius: 0;
+    width: initial;
+    height: initial;
+}
+
+.GraphSubWindow.windowed > .resizer {
+    display: none;
+}
+
+.GraphSubWindow:selected {
+    border-color: #44C0FF;
+}
+
+.GraphSubWindow > .mainContainer {
+    flex-direction: column;
+    align-items: stretch;
+}
+
+.GraphSubWindow.scrollable > .mainContainer {
+    position: absolute;
+    top:0;
+    left:0;
+    right:0;
+    bottom:0;
+}
+
+.GraphSubWindow > .mainContainer > #content {
+    flex-direction: column;
+    align-items: stretch;
+}
+
+.GraphSubWindow.scrollable > .mainContainer > #content {
+    position: absolute;
+    top:0;
+    left:0;
+    right:0;
+    bottom:0;
+    flex-direction: column;
+    align-items: stretch;
+}
+
+.GraphSubWindow > .mainContainer > #content > ScrollView {
+    flex: 1 0 0;
+}
+
+.GraphSubWindow > .mainContainer > #content > #contentContainer {
+    min-height: auto;
+    padding-left: 0;
+    padding-top: 0;
+    padding-right: 0;
+    padding-bottom: 6px;
+    flex-direction: column;
+    flex-grow: 1;
+    align-items: stretch;
+}
+.GraphSubWindow > #content > #header {
+    font-size: 15px;
+}
+
+.GraphSubWindow > .mainContainer > #content > #header {
+    flex-direction: row;
+    align-items: stretch;
+    background-color: #393939;
+    border-bottom-width: 1px;
+    border-color: #212121;
+    border-top-right-radius: 4px;
+    border-top-left-radius: 4px;
+    padding-left: 1px;
+    padding-top: 4px;
+    padding-bottom: 2px;
+}
+
+.GraphSubWindow.windowed > .mainContainer > #content > #header {
+    border-top-right-radius: 0;
+    border-top-left-radius: 0;
+}
+
+.GraphSubWindow > .mainContainer > #content > #header > #labelContainer {
+    flex: 1 0 0;
+    flex-direction: column;
+    align-items: stretch;
+}
+
+.GraphSubWindow > .mainContainer > #content > #header > #labelContainer > #titleLabel {
+    font-size : 14px;
+    color: #c1c1c1;
+}
+
+.GraphSubWindow > .mainContainer > #content > #header > #labelContainer > #subTitleLabel {
```

Comment:
Do all these selectors need to be this long? Did I already comment about this? Github may have not saved it so ignore if you've already responded.

Reply:
You did already comment about this, thanks for mentioning it again! Dropped out of my head, the selectors are simplified now, didn't realize I could just directly specify the element name if it was unique.